src/docs/product/sentry-basics/search/index.mdx

```diff
@@ -37,6 +37,12 @@ The tokens `is:resolved` and `user.username:"Jane Doe"` are standard search toke
 
 The token `example error` is utilizing the optional raw search and is passed as part of the issue search query (which uses a CONTAINS match similar to SQL). When using the optional raw search, you can provide _one_ string, and the query uses that entire string.
 
+<Note>
+<markdown>
+  Search for Releases only supports raw text and not query syntax.
```

Comment:
We may  have a link on the word Releases: https://docs.sentry.io/product/releases/

Reply:
Great suggestion. Making the change now.

Reply:
Had to remove the Note tag because it doesn't support linking